azkaban-common/src/main/java/azkaban/executor/ExecutorHealthChecker.java

```diff
@@ -136,14 +136,17 @@ void checkExecutorHealth() {
       Optional<ExecutorManagerException> healthcheckException = Optional.empty();
       Map<String, Object> results = null;
       try {
-        // Todo jamiesjc: add metrics to monitor the http call return time
+        long pingTime = System.currentTimeMillis();
         results = this.apiGateway
             .callWithExecutionId(executor.getHost(), executor.getPort(),
                 ConnectorParams.PING_ACTION, null, null, null);
+        pingTime = System.currentTimeMillis() - pingTime;
+        logger.info("Got ping response from " + executorDetailString(executor)
```

Comment:
Maybe log this in a `finally` block (line 151) so that we can can get this info for failed cases as well ?

Reply:
Ack. I shall do that.

Reply:
Is it possible to log action and executor details before the call? That way if the call hangs we can figure out the bad executor.